.vscode/extensions.json

```diff
@@ -0,0 +1,10 @@
+{
+	// See http://go.microsoft.com/fwlink/?LinkId=827846
+	// for the documentation about the extensions.json format
+	"recommendations": [
+		"christian-kohler.path-intellisense",
+		"christian-kohler.npm-intellisense",
+		"dbaeumer.vscode-eslint",
+		"flowtype.flow-for-vscode"
+	]
+}
```

Comment:
JS-y recommendations for extensions, Flow is used for internal typings but isn't "active" on the project, will just offer some useful autocompletions via the default type system